### PR TITLE
Ran all python code through black, code formater

### PR DESCRIPTION
**Pull request by de-sh**
Fixes un-formatted code with black

### Checklist
- [x] My branch is up-to-date with upstream/develop branch.
- [x] Everything works and tested for Python 3.5.2 and above.

### Description

The Black code formatter library is a python standard for maintaining readable code bases, I have run the code of the parser through it to make it more readable and PEP8 compliant.

### Change logs
- Code formatted with black

### CODE REVIEW

File: hydra_openapi_parser/openapi_parser.py [hunk 1]
```diff
@@ -103,7 +107,7 @@ def valid_endpoint(path: str) -> str:
     :return:
     """
     # "collection" or true means valid
-    path_ = path.split('/')
+    path_ = path.split("/")
```
Comment (Mec-iS) [inline hydra_openapi_parser/openapi_parser.py:110]: you should be sure that the rules for the formatter stick to the conventions we established for the codebase. For example: we decide to always use `'` single quotes for strings, except that for dictionaries meant to be dumped into JSON strings and for variables added via f-strings like `f'Some text with a {dict["variable"]}'`

In general you have to stick to how is done in hydrus.

Comment (Mec-iS) [reply hydra_openapi_parser/openapi_parser.py:110]: https://www.python.org/dev/peps/pep-0008/#string-quotes

**Review by Mec-iS — CHANGES_REQUESTED** [review]

File: samples/hydra_doc_sample.py [hunk 2]
```diff
@@ -104,50 +77,50 @@
...
+                    "writeonly": "true",
                 },
                 {
                     "@type": "SupportedProperty",
                     "property": "",
                     "readonly": "true",
                     "required": "false",
                     "title": "category",
-                    "writeonly": "true"
+                    "writeonly": "true",
```
Comment (Mec-iS) [inline samples/hydra_doc_sample.py:88]: I don't see the point of adding all these commas.

Please stick to [pep8 package](https://pypi.org/project/pep8/) as a reference libary

**Review by Mec-iS — CHANGES_REQUESTED** [review]

**Comment by Mec-iS** [comment]
See my review. We cannot really change the styling library every time, we have always been using [pep8 package](https://pypi.org/project/pep8/) as far as I remember

**Comment by de-sh** [comment]
> See my review. We cannot really change the styling library every time, we have always been using pep8 package as far as I remember.

Ok, These were a suggestion based on the output of [black](https://github.com/python/black), put it up for review, seems like this is not necessary... comments?

The PR can be closed if it seems to be unnecessary :+1:

**Comment by chrizandr** [comment]
I agree, we already have PEP8 based code formatting in all other repos. It would make more sense to stick to it than to change the format to something new. I don't potentially see any benefit in switching over to black.

EDIT: If you can give some insights as to why you think this is better, we might be open to more discussion about this. Otherwise I will close the PR.

**Comment by de-sh** [comment]
@chrizandr I thought [python/black](https://github.com/python/black) is provided by the PSF and hence using it would be better. There is no other reasoning and I have since come to think of not using a no compromise code formatter. I believe it's better to close this PR

**Pull request closed without merging**